src/main/java/net/dries007/tfc/client/render/blockentity/WindmillBlockEntityRenderer.java

```diff
@@ -31,13 +33,20 @@
 public class WindmillBlockEntityRenderer implements BlockEntityRenderer<WindmillBlockEntity>
 {
     public static final ResourceLocation BLADE_TEXTURE = Helpers.identifier("textures/entity/misc/windmill_blade.png");
+    public static final ResourceLocation LATTICE_BLADE_TEXTURE = Helpers.identifier("textures/entity/misc/windmill_blade_lattice.png");
+
+    public static final ResourceLocation RUSTIC_BLADE_TEXTURE = Helpers.identifier("textures/entity/misc/windmill_blade_rustic.png");
     private static final float[] NO_COLOR = {1f, 1f, 1f, 1f};
 
     private final WindmillBladeModel blade;
+    private final WindmillBladeLatticeModel blade_lattice;
+    private final WindmillBladeRusticModel blade_rustic;
```

Comment:
```suggestion
    private final WindmillBladeLatticeModel bladeLattice;
    private final WindmillBladeRusticModel bladeRustic;
```
as per java convention, non-static fields should be `lowerCamelCase`